acceptance/openstack/compute/v2/servers_test.go

```diff
@@ -105,6 +107,7 @@ func TestServersWithExtensionsCreateDestroy(t *testing.T) {
 	th.AssertEquals(t, int(extendedServer.PowerState), extendedstatus.RUNNING)
 	th.AssertEquals(t, extendedServer.TaskState, "")
 	th.AssertEquals(t, extendedServer.VmState, "active")
+	th.AssertEquals(t, extendedServer.LaunchedAt, extendedServer.Updated)
```

Comment:
The first time I ran this, it worked, but on the second attempt, these two values were slightly different:

```
"OS-SRV-USG:launched_at": "2018-07-28T02:17:57.000000"
"updated": "2018-07-28T02:18:02Z"
```

How about we check to see if `LaunchedAt` does not equal a zero-value `time.Time`?